Applications/FileManager/DirectoryView.h

```diff
@@ -36,6 +39,9 @@ class DirectoryView final : public GStackWidget {
     ViewMode m_view_mode { Invalid };
 
     Retained<GDirectoryModel> m_model;
+    int path_history_position;
```

Comment:
style nit: m_ prefix on members - and it might be a good idea to initialize the int in the header too for consistency:

    int m_path_history_position { 0 };